Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
```

Comment:
Try this syntax for views, when the button is first accessed (usually to add it to the view) this closure is called which returns the setup button. It's only called once, from then on it's just the button that it originally returned 

Private var button: UIButton =  {
Let button = UIButton()
//configure button as needed but don't do the layout here

return button
}))

Reply:
Thanks for your help on this this evening. I do have one question about what is deemed 'layout' exactly...? 

For example, when the closure is called you'll see I've included setting the title, background colour and then adding the target. 

I've then called the corner radius and the TAMIC in the layout method. 

Is that accurate? Or should I have put the corner radius and TAMIC in the button closure?

Reply:
The corner radius can only be set once the view has laid out, otherwise the height could be incorrect. It ideally needs to be called in viewDidLayoutSubviews so we know it's actually been laid out